bgpd/bgp_evpn_vty.c

```diff
@@ -1030,13 +1049,37 @@ static void show_vni_entry(struct hash_bucket *bucket, void *args[])
 		json_object_string_add(json_vni, "type", "L2");
 		json_object_string_add(json_vni, "inKernel",
 				       is_vni_live(vpn) ? "True" : "False");
-		json_object_string_add(json_vni, "originatorIp",
-				       inet_ntoa(vpn->originator_ip));
-		json_object_string_add(json_vni, "originatorIp",
-				       inet_ntoa(vpn->originator_ip));
 		json_object_string_add(
 			json_vni, "rd",
 			prefix_rd2str(&vpn->prd, buf2, sizeof(buf2)));
+		json_object_string_add(json_vni, "originatorIp",
+				       inet_ntoa(vpn->originator_ip));
+		json_object_string_add(json_vni, "mcastGroup",
+				       inet_ntoa(vpn->mcast_grp));
+		/* per vni knob is enabled -- Enabled
+		 * Global knob is enabled  -- Active
+		 * default  -- Disabled
+		 */
+		if (!vpn->advertise_gw_macip && bgp_evpn
+		    && bgp_evpn->advertise_gw_macip)
+			json_object_string_add(
+				json_vni, "advertiseGatewayMacip", "Active");
```

Comment:
here the same: `advertiseGatewayMacip` => `advertiseGatewayMacIp`?

Reply:
advertiseGatewayMacip is existing field and if changed it breaks parsing scripts around it. I suggest either we leave it as it is or add a new one (additional) advertiseGatewayMacIp.

Reply:
@chiragshah6 on this note, one field was changed by this PR as shown below

	- json_object_string_add(json, "kernelFlag", "Yes");
	+ json_object_string_add(json, "inKernel", "True");

I ignored this, as not many may be using this flag. Otherwise, we need to keep the old one intact and add the new one to maintain compatibility with old json output.